src/components/BorrowerProfile/BorrowerName.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+	<h1 class="text-h2">
+		{{ name }}
```

Comment:
Should this just be an <h2> ?

Reply:
@BoulderBrains In an earlier discussion with @emuvente and @ryan-ludwig, it was recommended that i do an h1 with a class of h2